radicle-node/src/service.rs

```diff
@@ -568,12 +568,14 @@ where
                 if other == rid {
                     debug!(target: "service", "Ignoring redundant attempt to fetch {rid} from {from}");
                 } else {
-                    // TODO: If we can't fetch, it's because we're already fetching from
-                    // this peer. So we need to queue the request, or find another peer.
-                    error!(
+                    // If we can't fetch, it's because we're already fetching from
+                    // this peer. So we need to queue the request.
+                    // TODO: consider to find another peer.
+                    debug!(
                         target: "service",
-                        "Dropping fetch for {rid} from {from}: another fetch is ongoing"
+                        "Queuing fetch for {rid} from {from}: another fetch is ongoing"
```

Comment:
```suggestion
                        "Queueing fetch for {rid} from {from}: another fetch is ongoing"
```
I think?

Reply:
✅

Reply:
Will update.   It's interesting what chatGPT said:

```
Both "queuing" and "queueing" are correct spellings of the word, with "queuing" being the more common spelling in British English and "queueing" being more common in American English. However, both spellings are widely accepted and can be used interchangeably.
```

Reply:
Oh interesting!